tests/queries/0_stateless/03254_attach_part_order.sql

```diff
@@ -0,0 +1,35 @@
+DROP DATABASE IF EXISTS test_attach_order_db;
+CREATE DATABASE test_attach_order_db ENGINE=Atomic;
+
+CREATE TABLE test_attach_order_db.test_table
+(
+    dt DateTime,
+    id UInt32,
+    url String,
+    visits UInt32
+)
+ENGINE ReplacingMergeTree
+ORDER BY (dt, id)
+PARTITION BY toYYYYMM(dt);
+
```

Comment:
You need to add `system stop merges test_attach_order_db.test_table;` to make it stable https://fiddle.clickhouse.com/e0d55ee1-32af-4d1c-aca1-b8088a8f4aa6

Reply:
Also it's better to remove a usage of all `database` (test_attach_order_db) because tests are run in parallel and each test uses its own database. Otherwise you need to add a tag `-- Tags: no-parallel`

Reply:
thanks